content/docs/examples/advanced-gateways/ingress-certmgr/index.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Securing Kubernetes Ingress with Cert-Manager
+description: Demonstrates how to obtain "Let's Encrypt" TLS certificates for Kubernetes Ingress automatically using Cert-Manager.
+weight: 70
+keywords: [traffic-management,ingress,https,cert-manager,acme,sds]
+---
+
+This tutorial demonstrates the use of Istio as a secure Kubernetes Ingress controller with TLS certificates issued by "Let's Encrypt". We will start with a clean Istio installation, create an example service, expose it using the Kubernetes `Ingress` resource and get it secured by instructing `cert-manager` (bundled with Istio) to manage issuance and renewal of TLS certificates that will be further delivered to the Istio Ingress Gateway and hot-swapped as necessary via the means of SDS (Secrets Discovery Service).
```

Comment:
is "Kubernetes `INgress` right?  We just deprecated this.

Reply:
Yes, it is. Looks like the current consensus is that we are keeping K8s Ingress and dropping the legacy "ingress" from the installer (I'm working on a PR). This example demonstates k8s ingress support via the ingressgateway, i.e. legacy "ingress" is not required and we can definitely clean that up.